app/graphql/types/student_type.rb

```diff
@@ -25,6 +26,16 @@ def avatar_url
         end
     end
 
+    def last_seen_at
+      BatchLoader::GraphQL
+        .for(object.user_id)
+        .batch do |user_ids, loader|
+          User
+            .where(id: user_ids)
+            .each { |user| loader.call(user.id, user.last_seen_at) }
```

Comment:
> so should we show nothing there? or "Last seen unknown"?

@arpancodes I think it might be smart to update the following line to check `user.current_sign_in_at` if required:

```suggestion
            .each { |user| loader.call(user.id, user.last_seen_at || user.current_sign_in_at) }
```

Assuming that we consider `current_sign_in_at` as well, all users who have signed in at least once should have a value sent to the UI.

The UI seems to have enough space here, so this means that we could say something like _"This student has never signed in"_...